### ISSUE TITLE
No data for Greece, not even the countries location.

### ISSUE THREAD

**Issue by redman684** [CSSEGISandData/COVID-19#561]
In the list of countries, is has a count for Greece, but when clicking it, you get 'no data' in the various windows. Even the map does not focus on Greece.

**Comment by Bost** [comment]
What about providing a direct link so that the people who actually will fix it have it easier? This repo has 227 opened issues. It's not like they're being bored.

**Comment by redman684** [comment]
A link to what? It's a single page and you have to click the "Greece" in the list of countries, which is not a link.

**Comment by SchmidL** [comment]
For me it works! (Windows 10, Chrome) What browser you're using?

**Comment by hristo-mavrodiev** [comment]
It's partly working, depending on the browser/ system
![image](https://user-images.githubusercontent.com/61916767/76519716-2b0ffb00-646a-11ea-8d81-756b40f79b8e.png)
On Win10 v1909 with Chrome Version 80.0.3987.132 (Official Build) (64-bit) It's working fine.
On Win10 1909, with Internet Explorer 11, also working fine.
On Debian 10, with Chromium and Firefox there is an issue with the visualization.

**Comment by redman684** [comment]
Firefox (latest) and Seamonkey (latest) on Ubuntu 18.
I can't reproduce it now, not even on the window I had open (and did not reload) when/sinds reporting this. This is not a one time occurrence, I've seen it happen multiple times over the past week, always with Greece and only with Greece. I'm now seeing Greece correctly for the first time.
Maybe some request fails and gets cached for some time. I'll keep the debugger open for a few days, let's see if that sheds some light on things.

**Comment by Bost** [comment]
> A link to what? It's a single page and you have to click the "Greece" in the list of countries, which is not a link.

A link to have an evidence. And to save people's time. Otherwise only God knows what exactly you've been doing.

> For me it works! (Windows 10, Chrome) What browser you're using?

You see. Without the evidence your bug report looks more like a PEBKAC than a bug.

**Comment by Bost** [comment]
> I can't reproduce it now,

Ah, here we have it :) LMAO

**Comment by redman684** [comment]
> A link to have an evidence. And to save people's time. Otherwise only God knows what exactly you've been doing.

I've only seen it go wrong on 2 different browsers for the past week. I was not aware it ever worked.

**Comment by Bost** [comment]
> > A link to have an evidence. And to save people's time. Otherwise only God knows what exactly you've been doing.
> 
> I've only seen it go wrong on 2 different browsers for the past week. I was not aware it ever worked.

Yeah yeah. Me too, I've seen things, too. But that's OK, we're all friends here :)

**Comment by Alexandros-Kordatzakis** [comment]
That's happening when they update the data. Happens all the time. And not only for Greece.

I would suggest you close this issue. They know about that, but it's not a "bug".

**Comment by Bost** [comment]
> That's happening when they update the data. Happens all the time. And not only for Greece.

Again, that's just a claim. That's not how engineers and developers work. Those guys (i.e. us) need evidence - logfiles, numbers, screenshots whatever. Talk is cheap show us data. Thanks.

**Comment by Bost** [comment]
> I would suggest you close this issue.

Well there's some piece of evidence from @hristo-mavrodiev 

> On Win10 v1909 with Chrome Version 80.0.3987.132 (Official Build) (64-bit) It's working fine.
> On Win10 1909, with Internet Explorer 11, also working fine.
> On Debian 10, with Chromium and Firefox there is an issue with the visualization.

Keep the bug, open. It looks like there will be some proper evidence, eventually.

**Comment by Bost** [comment]
> They know about that, but it's not a "bug".

Well then, it's not a bug it is feature? SRSLY? If they know about it then there's already a bug report and this one is its duplicate. Right? So: go and find it please. And link it to this one. Thank you (on behalf of humanity, hehe. )

**Comment by hristo-mavrodiev** [comment]
I've just updated everything possible with :
```sudo apt-get upgrade```
After the upgrade the the issue can't be reproduced.
I was using Firefox 68.2 updated to 68.5.0esr (64-bit) on Debian 10.
My opinion is that this was a browser/ArcGIS platform issue, no related with the data and repository.
